tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -3017,3 +3018,170 @@ def call(self, inputs, state):
 
       new_state = rnn_cell_impl.LSTMStateTuple(new_c, new_h)
       return new_h, new_state
+
+
+class NLSTMCell(rnn_cell_impl.RNNCell):
+  """Nested LSTM Cell. Adapted from `rnn_cell_impl.LSTMCell`
+
+  The implementation is based on:
+    https://arxiv.org/abs/1801.10308
+    JRA. Moniz, D. Krueger.
+    "Nested LSTMs"
+    ACML, PMLR 77:530-544, 2017
+  """
+
+  def __init__(self, num_units, depth, forget_bias=1.0,
```

Comment:
Should there be an additional argument `use_bias`?

Reply:
Agree. Done!